.github/workflows/build.yml

```diff
@@ -31,16 +31,17 @@ jobs:
         key: v3-${{ runner.os }}-${{ hashFiles('pmdtester.gemspec') }}
         restore-keys: |
           v3-${{ runner.os }}-
-    - name: Set up Ruby 2.7
+    - name: Set up Ruby 3.3
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.7
+        ruby-version: 3.3
     - name: Setup Environment
       shell: bash
       run: |
         echo "LANG=en_US.UTF-8" >> $GITHUB_ENV
         echo "MAVEN_OPTS=-Dmaven.wagon.httpconnectionManager.ttlSeconds=180 -Dmaven.wagon.http.retryHandler.count=3" >> $GITHUB_ENV
-        echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/pmd/build-tools/17/scripts" >> $GITHUB_ENV
+        #echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/pmd/build-tools/master/scripts" >> $GITHUB_ENV
+        echo "PMD_CI_SCRIPTS_URL=https://raw.githubusercontent.com/adangel/build-tools/ruby3/scripts" >> $GITHUB_ENV
```

Comment:
if we merge https://github.com/pmd/build-tools/pull/29 before I don't think we need this…